internal/payloads/reservation_payload.go

```diff
@@ -42,6 +42,10 @@ type GenericReservationResponsePayload struct {
 	Success *bool `json:"success" nullable:"true"`
 }
 
+type InstanceResponse struct {
+	*models.ReservationInstance
+}
```

Comment:
Please do not use embedding here since it also pulls `instance_id` unwanted field (makes no sense to expose it through the API). Just copy the fields and in the copy function copy one by one.

Reply:
why the instance_id is unwanted?

Reply:
I'd agree with amir here, we are exposing the instance_id in the UI, so we should include it.
I'd back the copying is bit better because its bit more clear what data are we exposing.
I believe the field we don't want is `reservation_id`, as that's already included in the response.